sea-orm-rocket/lib/Cargo.toml

```diff
@@ -24,3 +24,7 @@ version = "0.5.0-rc.1"
 version = "0.5.0-rc.1"
 default-features = false
 features = ["json"]
+
+[dependencies.rocket_okapi]
```

Comment:
I think @HigherOrderLogic means that this should be an optional cargo feature.

Reply:
@tyt2y3 I made this dependency optional.